package.json

```diff
@@ -26,7 +26,7 @@
     "build": "npx cross-env BROWSER=none && rollup -c",
     "clean": "rm -rf dist && rm -f ucp-npm-*.tgz",
     "pack": "npm run clean && npm run build && npm pack",
-    "test": "echo \"Error: no test specified\" && exit 0"
+    "test": "echo \"No test specified\" && exit 0"
```

Comment:
Connect team is redoing all tests in the Isolate Widget project. Once we are ready to migrate to that codebase, we will have all their tests.